graphql/chat/mutations.ts

```diff
@@ -65,11 +66,14 @@ export class MutateChatResolver {
             subject: subcourse.course.name,
             custom: {
                 start: subcourse.lecture[0].start.toISOString(),
-                type: 'course',
+                type: groupChatType === ChatType.ANNOUNCEMENT ? 'announcement' : 'course',
```

Comment:
Why are we using different enums and values in similar places? I so far saw:
- announcement
- course
- normal
What's the matter with those types?

Reply:
https://github.com/corona-school/backend/pull/686